src/_content/posts/introducing-keynote-natalia-bidart.md

```diff
@@ -0,0 +1,52 @@
+
+>The fellows play an essential role in managing and unifying the diverse contributions that come from the community. We ensure consistency, maintain high standards, and safeguard the overall quality of Django. You can think of our role as being the housekeepers of Django, putting up guardrails to maintain the framework's integrity.
+
+>It’s important to raise awareness about the fellowship, not just for fundraising, but also because many people, even those who use Django daily, aren’t fully aware of what the program entails. When I was appointed a fellow, I found myself explaining what the Django Fellowship is to my peers, some of whom teased, "Is it something like the Fellowship of the Ring?"
+
+## What are you most looking forward to at DjangoCon US 2024?
+
+>This will be my third DjangoCon. Last year was my first, and I also attended DjangoCon Europe earlier this year. What's exciting about this DjangoCon US is that both Sarah Boyce and I, the current Django fellows, will be there together for the first time. Last year, my fellow, Marius, couldn't attend, and Sarah couldn't make it to DjangoCon Europe because she was getting married. So, having both of us there is something I'm really looking forward to.
+
+>This time around, I feel more prepared and less nervous. Last year, I was anxious and had a lot of questions about how things would work. Now that I have some answers and more experience, I hope to enjoy the conference even more. If I can get my talk ready on time, I think I’ll be able to relax and fully immerse myself in the event. Last year was more intense for me because it was my first DjangoCon as a fellow, and I wasn't sure what to expect or what people expected from me.
+
+## What are your favorite topics at conferences?
+
+>I enjoy non-technical topics, and I find that techical topics can be tackled effectively through a computer. Since they are often algorithmic or complex, I can engage with them directly at the keyboard. However, when it comes to topics like inclusion, I believe they are much more nuanced and multifaceted. These discussions benefit greatly from a conference setting where the environment is more fluid, and you can pick up on non-verbal cues like gestures and tones of voice, which are crucial for understanding the full context.
+
+>I think it's essential to acknowledge that Django’s progress depends on the strength and diversity of its community. You can't afford to be selective about who is welcome in the community based on arbitrary criteria. Of course, there are standards of conduct that everyone should adhere to, and that’s necessary for maintaining a healthy and inclusive environment. But beyond that, openness and inclusivity are vital for the community to thrive.
+
+
+## What do you like to do outside of your normal work day?
```

Comment:
```suggestion
## What do you like to do outside of your normal workday?
```

Reply:
You can go either way on this one I think